package.json

```diff
@@ -35,6 +35,7 @@
     "lodash": "4.6.1",
     "mkdirp": "0.5.1",
     "moment": "2.12.0",
+    "path-is-inside": "1.0.1",
```

Comment:
We can use [path.basename](https://nodejs.org/api/path.html#path_path_basename_path_ext) to fix writing outside.